cms/static/cms/js/require-config.js

```diff
@@ -63,10 +63,13 @@
             "xblock": "coffee/src/xblock",
             "utility": "js/src/utility",
             "accessibility": "js/src/accessibility_tools",
-            "draggabilly": "js/vendor/draggabilly.pkgd",
             "URI": "js/vendor/URI.min",
             "ieshim": "js/src/ie_shim",
             "tooltip_manager": "js/src/tooltip_manager",
+            "modernizr": "js/vendor/afontgarde/modernizr.fontface-generatedcontent",
```

Comment:
If the one with the library has both "fontface" and "generated-content" support, it should work for A Font Garde as well and we would be able to use it instead.